bftengine/tests/simpleTest/persistency_test.cpp

```diff
@@ -0,0 +1,99 @@
+//
+// Created by Igor Golikov on 2019-07-01.
```

Comment:
This is information that can be approximately be discovered via git. Feel free to leave it in though. 

We should also have a VMware copyright header in this file.